workflow-runtime/dependencies/jvmRuntimeClasspath.txt

```diff
@@ -0,0 +1,9 @@
+:workflow-core
+com.squareup.okio:okio-jvm:3.0.0
+com.squareup.okio:okio:3.0.0
+org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.6.10
```

Comment:
Does not have the kotlin-bom dependency compared to `workflow-runtime/dependencies/runtimeClasspath.txt` 🤔